### DISCUSSION TITLE
How to Filter Translatable Field

### DISCUSSION THREAD

**Discussion by ibrahim-999** [laravel/nova-issues#4324] [Q&A] [answered]
- Laravel Version: v8.83.10
- Nova Version: 3.32.0
- PHP Version: PHP 7.4
- Browser type and version: Chrome 100.0
- Database Driver & Version:Mysql 5
- Operating System and Version: linux mint
### Description:
i want to filter a resource according to title wether English or other language any help

### Detailed
    this is the function apply how can i put condition to this query so i can choose between title->en and title->es 

    public function apply(Request $request, $query, $value)
    {
        return $query->whereNotNull('title->en');
    }

    public function options(Request $request)
    {
        return [
            'ES' => 'title->es',
            'En' => 'title->en'
        ];
    }

**Comment by crynobone** [comment, marked as answer]
Shouldn't the following just work?

```php

public function apply(Request $request, $query, $value)
{
    return $query->whereNotNull($value);
}

public function options(Request $request)
{
    return [
        'ES' => 'title->es',
        'En' => 'title->en'
    ];
}
